src/index.ts

```diff
@@ -170,19 +170,27 @@ async function buildSingleFile(
 		);
 	}
 
+	// The compiled binary in Windows has the `.exe` extension
+	const binExtension = (process.platform === "win32") ? '.exe' : "";
+
 	const bin = path.join(
 		path.dirname(cargoTomlFile),
 		"target",
 		builderDebug ? "debug" : "release",
-		binName
+		binName + binExtension
 	);
 
+	debug("Binary file is: " + bin);
+	console.log("Binary file is: " + bin);
```

Comment:
Here's an extra `console.log` that should be removed

Reply:
Oops...
Fixed it!